mlir/lib/Dialect/Rock/IR/RockDialect.cpp

```diff
@@ -489,14 +501,14 @@ GemmSize GemmSize::fromConvolution(ConvOpType type,
   case ConvOpType::Fwd:
     gemmGSize = sizes.g;
     gemmMSize = sizes.k;
-    gemmKSize = sizes.c * sizes.y * sizes.x;
-    gemmNSize = sizes.n * sizes.ho * sizes.wo;
+    gemmKSize = sizes.c * sizes.fil[0] * sizes.fil[1];
```

Comment:
References to arrays now.  Not sure how some of these expressions should be extended for 3-D.

Reply:
These expressions extend naturally - they're the product of the array.